pyproject.toml

```diff
@@ -52,10 +52,19 @@ typecheck = [
 codegen = [
     "libcst>=1.5.1,<2",
 ]
+tools = [
+    "requests~=2.0",
+    "tomli>=2.2.1 ; python_full_version < '3.12'",
+]
```

Comment:
Since it's a standalone script, [PEP 722](https://peps.python.org/pep-0722/) might be more appropriate. It's even supported by `uv` (https://github.com/astral-sh/uv/issues/7396) by running `uv run --script tools/<long name>.py`

Reply:
I ended up moving to `importlib`, as you suggested above. That makes the whole script rely on standard libraries, freeing up the script requirement. Also, it needs to be run in the main env, as `importlib` needs to fetch the currently installed `scipy`.